DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -62,9 +62,12 @@ void pat::PackedCandidate::packVtx(bool unpackAfterwards) {
 }
 
 void pat::PackedCandidate::unpack() const {
-    p4_ = PolarLorentzVector(MiniFloatConverter::float16to32(packedPt_),
+    float pt = MiniFloatConverter::float16to32(packedPt_);
+    double shift = (pt<1. ? 0.1*pt : 0.1/pt); // shift particle phi to break degeneracies in angular separations
+    double phi = int16_t(packedPhi_)*3.2f/std::numeric_limits<int16_t>::max() + shift*3.2/std::numeric_limits<int16_t>::max();
```

Comment:
When I suggested to move the degridification to packing, I meant to do it so that after one pack->unpack cycle, additional pack->unpack keeps unpacked values unchanged.